docs/styles.md

```diff
@@ -26,16 +26,20 @@
 * Layout
   - Flex
     - Initialization of Flex Box
-      - [Required Flex Class](#side-by-side-os-raise-d-flex)
+      - [Required Flex Class](#side-by-side-os-raise-d-flex-nowrap)
     - Content Positioning
-      - [Center Side-by-Side Content Vertically](#side-by-side-os-raise-d-flex)
-      - [Center Side-by-Side Content Horizontally](#side-by-side-os-raise-d-flex)
-      - [Add Space Between Side-by-Side Content](#side-by-side-os-raise-d-flex)
-      - [Add Space Equally Around Side-by-Side Content](#side-by-side-os-raise-d-flex)
+      - [Center Side-by-Side Content Vertically](#side-by-side-os-raise-d-flex-nowrap)
+      - [Center Side-by-Side Content Horizontally](#side-by-side-os-raise-d-flex-nowrap)
+      - [Add Space Between Side-by-Side Content](#side-by-side-os-raise-d-flex-nowrap)
+      - [Add Space Equally Around Side-by-Side Content](#side-by-side-os-raise-d-flex-nowrap)
+    - Content Spacing
+      - [Add 16px of Space Between Content](#side-by-side-os-raise-d-flex-nowrap)
+      - [Add 32px of Space Between Content](#side-by-side-os-raise-d-flex-nowrap)
 * Components
   - [Gray Box](#gray-box-os-raise-graybox)
```

Comment:
This isn't directly related to your PR, but maybe something we can fix real quick: The actual location of the `Gray Box` content does not reflect what is in the ToC. Can we move it so it is adjacent to `Motivcational Content`, etc.?

Reply:
Yes, this can be done as well.

Reply:
Fixed.